libraries/radpdfprocessing/formats-and-conversion/pdf/pdfstreamwriter/pdffilesource.md

```diff
@@ -0,0 +1,55 @@
+---
+title: PdfFileSource
+page_title: PdfFileSource
+description: PdfFileSource
+slug: radpdfprocessing-formats-and-conversion-pdf-pdfstreamwriter-pdffilesource
+tags: pdf,stream,file,source
+published: True
+position: 4
+---
+
+# Overview
+
+The **PdfFileSource** class represents the content of an existing PDF file.
+
+## Using PdfFileSource
+
+### Create an Instance
+
```

Comment:
Using two headers without any content between them brings a visualization that is not very pleasant. A big header than a separator with a big margin between the headers and the separator and then another header. The space between the headers seems a bit empty.